src/containers/ContextProvider/reducer.js

```diff
@@ -0,0 +1,39 @@
+import { headerDefinitions } from '../../util/data';
+export const stateActions = Object.freeze({
+  updatePatient: 'update_patient',
+  updateSetting: 'update_setting', // {type, settingId, value}
```

Comment:
```suggestion
  updateSetting: 'update_setting',
```

Reply:
the comment should be fine, the type for dispatch isn't defined anywhere so this reference can be helpful

Reply:
oh I misunderstood, so the comment is here to indicate something like this in TypeScript (since this repo doesn't use TS):
```ts
type Action = {
  type: 'update_patient',
  value: Patient
} | {
  type: 'update_setting',
  settingId: string,
  value: string | number
} | {
  type: 'flag_startup',
  value: boolean
}
```

I'd rename the stateActions to actionTypes and leave comments beside updatePatient and flagStartup that look like the one next to updateSetting then